exercises/complex-numbers/.meta/generator/complex_numbers_case.rb

```diff
@@ -0,0 +1,36 @@
+require 'generator/exercise_case'
+
+class ComplexNumbersCase < Generator::ExerciseCase
+
+  def workload
+    indent_text(4, testcase)
+  end
+
+  def number
+    parse_number((respond_to? :input) ? input : z1)
```

Comment:
I'm not a fan of `z1` and `z2` as variable names.
Are these standard complex number terms?

Reply:
`z` is a standard complex value as `x` is a real one and `n` an integer one.
Here test specification uses either `input` or `z1` and `z2`.

Reply:
I figured this was canonical-data related. It's probably OK to stick with those names then.